app/views/events/overview.html.erb

```diff
@@ -14,8 +14,20 @@
   <% @event.teams.each do |team| %>
       <tr>
         <td><%= link_to team.name, team_path(team) %></td>
-        <td><%= @event.standing_of team %></td>
+        <td>
+          <% match = @event.last_match_of(team) %>
+          <% if match %>
+            <%= link_to @event.standing_of(team), match_path(@event.last_match_of(team)) %>
+            <% opponent = match.opponent_of(team) %>
+            <% if opponent.is_a? Team %>
+              gegen
+              <%= link_to opponent.name, team_path(opponent) %>
```

Comment:
I would rather use the `participant_link` method here. Otherwise, the automatic single-user-team-objects will not get skipped

Reply:
(actually not sure anymore if this functionality is done in the `participant_link` function, but it should be placed there)